hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/SingleTransactionRecordBuilderImpl.java

```diff
@@ -283,6 +285,8 @@ public SingleTransactionRecord build() {
                 .assessedCustomFees(assessedCustomFees)
                 .automaticTokenAssociations(newAutomaticTokenAssociations)
                 .paidStakingRewards(paidStakingRewards)
+                // @todo(nickpoorman) Add token type to RecordBuilder for BlockStream use.
+                // .tokenType(tokenType);
```

Comment:
This needs a change to protobufs right ? Is that already done as part of a HIP ?

Reply:
May need to be done in future work, or may just be done in the messages that replace SingleTransactionRecord for BlockStream; hence the TODO for Nick.